modules/gapi/src/streaming/onevpl/accelerators/accel_policy_interface.hpp

```diff
@@ -0,0 +1,59 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef GAPI_STREAMING_ONEVPL_ACCELERATORS_ACCEL_POLICY_INTERFACE_HPP
+#define GAPI_STREAMING_ONEVPL_ACCELERATORS_ACCEL_POLICY_INTERFACE_HPP
```

Comment:
as @smirnov-alexey  mentioned before - it should take a form  `path/file_name` if i understood correctly
to do not conflict with potential similar file names on uper-level

Reply:
Looks like some other headers are simplified to just `file_name`, so it can be applied here. I think only public header should follow `path/file_name` guideline. Sorry, if I was mistaken